paasta_tools/metrics/metastatus_lib.py

```diff
@@ -856,6 +857,15 @@ def get_resource_utilization_by_grouping(
         for attribute_value, slaves in slave_groupings.items()
     }
 
+@time_cache(ttl=300)
+def get_kube_all_nodes(kube_client: KubeClient) -> Sequence[V1Node]:
+    nodes: Sequence[V1Node] = get_all_nodes(kube_client)
+    return nodes
+
+@time_cache(ttl=300)
+def get_kube_all_pods(kube_client: KubeClient) -> Sequence[V1Pod]:
```

Comment:
minor nit: can you call these functions `get_all_[pods|nodes]_cached` instead? also, maybe they'd be better off living in `kubernetes_tools` instead?

Reply:
That makes sense, thanks. I've moved these over.